Sources/MessagesViewController.swift

```diff
@@ -175,6 +176,25 @@ extension MessagesViewController: UICollectionViewDataSource {
 
 }
 
+// MARK: - UITextFieldDelegate Methods
+
+extension MessagesViewController: UITextViewDelegate {
```

Comment:
@iJasonWade Thank you for the PR but this is pretty brittle considering the current app uses "New Message" as text and not placeholder text etc. There are a few minor UI details like this that need to be cleaned up.

I also already have the delegate for `textInputView` set inside of `MessageInputBar`. I'm trying to keep things out of the `MessagesViewController` when possible 🤔

I would also prefer discussing PRs first. We recently made our Slack channel public for this reason. 
It looks like I should open an issue for this.